packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -96,12 +103,14 @@ const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HT
     isNonModal,
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     isDismissable,
+    overlayProps,
+    preventScroll,
     ...otherProps
   } = props;
-  let {overlayProps} = useOverlay({...props, isDismissable: isDismissable && isOpen}, ref);
```

Comment:
looks like some logic didn't move with the call to useOverlay?

Reply:
thanks, good catch